resources/public/api/conf/1.0/application.raml

```diff
@@ -550,13 +570,15 @@ traits:
         get:
           displayName: Get all messages relating to a Movement Departure
           description: |
-            Get all messages sent within 31 days of the goods being released relating to a Movement Departure. Any Movements that are more than 31 days old will have been archived. Archived notifications will be return an HTTP 404 Not Found status.
+            Retrieve all messages relating to a specific transit movement departure sent to the Customs Office of Departure within 31 days of its release at its final destination.
+
+            You can filter for departures received after a specific date and time.
 
-            You may opt to filter this list of messages so that you only receive messages that have been received since a specified date and time.
+            Movements older than 31 days are archived, so any messages to relating to them cannot be returned by the API.
```

Comment:
As above. Also I notice you've used 31 days old or younger in other places -- do we want to be consistent?

Reply:
It depends on the context. Agree that we need to use consistent wording where possible.